src/main/java/com/hundred/percent/capstone/Invoicify/company/DatabaseConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.hundred.percent.capstone.Invoicify.company;
```

Comment:
DatabaseConfig is generic class for whole service, So, please move this out of company package